go.mod

```diff
@@ -1,5 +1,8 @@
-module github.com/containerd/btrfs
+module github.com/containerd/btrfs/v2
 
-go 1.15
+go 1.16
 
-require github.com/pkg/errors v0.9.1
+require (
+	github.com/dennwc/ioctl v1.0.0
+	github.com/spf13/cobra v1.1.3
```

Comment:
Curious; do we use the command-line binary in containerd itself? If not, (e.g., only for testing) would it be possible to refactor it into  a separate (sub)module?

Asking, because I know Cobra comes with a lot of (indirect) dependencies, including (e.g.) specific versions of gRPC and various other things.

If possible, we should try to keep Cobra out of the dependency tree to prevent it becoming an issue.

Reply:
done

Reply:
great, thanks!